modules/nf-core/pairix/tests/main.nf.test

```diff
@@ -0,0 +1,32 @@
+
+nextflow_process {
+
+    name "Test Process PAIRIX"
+    script "../main.nf"
+    process "PAIRIX"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "pairix"
+
+    test("test-pairix") {
+        
+        when {
+            process {
+                """
+                input[0] = [ [ id:'test', single_end:false ], // meta map
+				file("https://raw.githubusercontent.com/4dn-dcic/pairix/master/samples/4dn.bsorted.chr21_22_only.nontriangle.pairs.gz", checkIfExists: true) ]
```

Comment:
I thought it was said to not have external github repo's linked for test-data and that they need to be added to nf-core/testdatasets. Pinging @jfy133 as I saw a message where he talks about the `test-data-task-force`

Reply:
Good point @Joon-Klaps 
This was an auto conversion from pyteat to nf-test. I am happy to put the data through the nf-core datasets repo.

Reply:
I would much rather focus on the pytest -> nf-test migration for now, and worry about the external links at a later point.